dev-tools/test.sh

```diff
@@ -16,7 +16,8 @@ require_database
 # Set dummy key to enable SUMM.AI during testing
 export INTEGREAT_CMS_SUMM_AI_API_KEY="dummy"
 
-while test $# -gt 0; do
+# Parse given commend line arguments
+while [[ $# -gt 0 ]]; do
```

Comment:
Since we're also using the extended test command in other places, we could also use it here for consistency reasons